rcp/org.testeditor.rcp4.tcltestrun.tests/src/org/testeditor/rcp4/tcltestrun/TclMavenLauncherTest.xtend

```diff
@@ -5,41 +5,38 @@ import java.nio.file.Files
 import org.eclipse.core.resources.IProject
 import org.eclipse.core.runtime.IPath
 import org.junit.Test
-import org.junit.runner.RunWith
 import org.mockito.InjectMocks
 import org.mockito.Mock
-import org.mockito.runners.MockitoJUnitRunner
+import org.testeditor.dsl.common.testing.AbstractTest
 
-import static org.junit.Assert.*
-import static org.mockito.Mockito.*
+import static extension org.mockito.Mockito.*
+import org.junit.Rule
+import org.junit.rules.TemporaryFolder
 
-@RunWith(MockitoJUnitRunner)
-class TclMavenLauncherTest {
+class TclMavenLauncherTest extends AbstractTest {
 
-	@Mock
-	MavenExecutor executor
-
-	@InjectMocks
-	TclMavenLauncher launcher
+	@InjectMocks TclMavenLauncher launcher
+	@Mock MavenExecutor executor
+	@Rule public TemporaryFolder tempFolder = new TemporaryFolder();
 
 	@Test
 	def void testGetProfiles() {
-		//given
-		val targetDir = new File(System.getProperty("java.io.tmpdir"),"target")
-		if(!targetDir.exists) {
+		// given
+		val targetDir = tempFolder.newFolder("target")
+		if (!targetDir.exists) {
 			targetDir.mkdir
 		}
-		Files.write(new File(targetDir,"profiles.txt").toPath,'''Profile Id: foo (Active: true)
+		Files.write(new File(targetDir, "profiles.txt").toPath, '''Profile Id: foo (Active: true)
```

Comment:
We should extract the contents into a separate variable, this line is quite hard to read. Also using Guava `com.google.common.io.Files.write(CharSequence, File, Charset)` is a better fit here.